fullstack-cert/python-projects/shortest-path-workshop/main.py

```diff
@@ -0,0 +1,55 @@
+nodes = ['A', 'B', 'C', 'D', 'E', 'F']
```

Comment:
Is it really necessary to label the nodes with letters? Can't we use their number to reference them?

Reply:
Letters help make the graph feel more concrete and relatable. Seeing `"A -> C -> D"` gives more context than `"0 -> 2 -> 3"`.